events/static/js/signup/components/Manage.js

```diff
@@ -118,6 +136,26 @@ class Manage extends Component {
       )
     }
 
+  
+
+    if (event.fee == 0){
```

Comment:
This is a small nitpick. When comparing floating point numbers, it's actually not safe to compare floating point numbers with "==". Floating points in memory are just approximations of the real "mathematical" fraction. Comparing "0f == 0f" will give false in certain environments.

I think in the Javascript case this will work, because it might parse 0.0 as 0 if `event.fee` is a float, but that's up to the javascript implementation.

The safest way to go about this is to always use inequality operators, in this case `event.fee <= 0`.

Reply:
thank you, done!